online-shop/src/app/product-list/product-list.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="shop-title">
+  <h2>Products</h2>
+</div>
+
+<div class="table-products">
+<table>
+  <tr class="table-headers">
+    <th>Name</th>
+    <th>Description</th>
+    <th>Price</th>
+  </tr>
+   <tr class="table-data" *ngFor="let product of products">
+     <td>{{product.name}}</td>
+     <td>{{product.description}}</td>
+     <td>{{product.price}}</td>
+     <button [routerLink]="['product', product.id]">More <br> details</button>
```

Comment:
using <br> for spacing is not the best thing to do.
Also, when you start setting parameters on navigation, is better to do it in the .ts file using the router.navigate
https://angular.io/guide/router